File: SothemaGoalManagement.API/Models/EvaluationFile.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using SothemaGoalManagement.API.Helpers;

namespace SothemaGoalManagement.API.Models
{
    public class EvaluationFile
    {
        public int Id { get; set; }

        public string Title { get; set; }

        public int Year { get; set; }

        public string Status { get; set; }

        public DateTime Created { get; set; }

        public User Owner { get; set; }

        public int OwnerId { get; set; }

        public ICollection<EvaluationFileBehavioralSkill> BehavioralSkills { get; set; }

        public int StrategyId { get; set; }

        public Strategy Strategy { get; set; }

        public bool Sealed { get; set; }

        public DateTime SealedDate { get; set; }


        public EvaluationFile()
        {
            BehavioralSkills = new Collection<EvaluationFileBehavioralSkill>();
        }
    }
}

File: SothemaGoalManagement-SPA/src/app/_models/evaluationFile.ts
import { BehavioralSkill } from './behavioralSkill';
import { Strategy } from "./strategy";
import { Axis } from './axis';

export interface EvaluationFile {
  id: number;
  title: string;
  year: number;
  strategy: Strategy;
  behavioralSkills: BehavioralSkill[];
  axisList: Axis[];
  createdByName: string;
  createdByNameId: number;
  status: string;
  created: Date;
  sealed?: boolean;
  sealedDate?: Date;
}
